named-entity-recognition/run_ner.py

```diff
@@ -222,15 +256,15 @@ def align_predictions(predictions: np.ndarray, label_ids: np.ndarray) -> Tuple[L
 
     def compute_metrics(p: EvalPrediction) -> Dict:
         preds_list, out_label_list = align_predictions(p.predictions, p.label_ids)
-        
+
         return {
             "precision": precision_score(out_label_list, preds_list),
             "recall": recall_score(out_label_list, preds_list),
             "f1": f1_score(out_label_list, preds_list),
         }
 
     # Initialize our Trainer
-    trainer = Trainer(
+    trainer = RMSPropTrainer(
```

Comment:
Let's leave the previous one as comment so it is easy to switch
```suggestion
    # trainer = Trainer(
    trainer = RMSPropTrainer(
```